src/components/carData/carData.java

```diff
@@ -0,0 +1,118 @@
+package components.carData;
+
+/**
+ * Represents a car and manages its information including make, model, year, and
+ * mileage.
+ */
+public class carData implements carDataKernel, carDataSecondary {
+    private String make;
```

Comment:
You might make this an enum. I know there are a lot of possible makes, so you may not prefer an enum. However, it will make your code easier to write.